shencha-master/shencha-master/test_api_all.py

```
@@ -1,161 +1,219 @@
+        for patent in response.valid_patents:
+            f.write(f"{patent}\n")
+        f.write("\n有效论文:\n")
+        for paper in response.valid_papers:
+            f.write(f"{paper}\n")
+        f.write("\n统计信息:\n")
+        f.write(f"{response.comparison_stats}\n")
+
+    print(f"有效性检查结果已保存到 {VALIDITY_RESULTS_FILE}")
+
+
+import logging
```

Ruff (0.12.2)

Redefinition of unused `logging` from line 24

Remove definition: `logging`

(F811)